test/RequiredParameterTest.php

```diff
@@ -26,7 +26,7 @@ class RequiredParameterTest extends TestCase
      */
     private $required;
 
-    public function setUp()
+    protected function setUp()
```

Comment:
According to the [PHPUnit fixtures reference](https://phpunit.readthedocs.io/en/7.5/fixtures.html?highlight=fixtures), it should be the `protected setUp` and `protected tearDown` methods, not `public`.